### DISCUSSION TITLE
Finding number of bounces experienced by a receiver

### DISCUSSION THREAD

**Discussion by Pranathi-18** [NVlabs/sionna#421] [Q&A]
Hello there!!!
In a scenario with multiple receivers, shouldn't every receiver experience different number of reflections? And if so, is there any way to get the number of reflections experienced by any given receiver? Though a path with a negative delay is a non-existent path, I was particularly interested in knowing the number of reflections experienced by an individual receiver.

Thank you.

**Comment by SebastianCa** [comment]
Hi @Pranathi-18,

You are right, each receiver position can indeed experience a different number of reflections. Diffuse reflections, in particular, can add many valid paths, although each path may carry only little energy.

You can find a visualization of this in the [Ray Tracing Tutorial Notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#Ray-Tracing-for-Radio-Propagation). This tutorial provides an example of how to access the number of paths experienced by each receiver.

I hope this helps!
